doc/mgr/orchestrator.rst

```diff
@@ -256,9 +220,29 @@ the id is the numeric OSD ID, for MDS services it is the file system name::
     ceph orch daemon status <type> <instance-name> [--refresh]
 
 
+.. _orchestrator-cli-cephfs:
+    
+Depoying CephFS
+===============
+
+In order to set up a :term:`CephFS`, simply execute::
+
+    ceph fs volume create <fs_name> <placement spec>
+    
+Where
+
+* ``name`` is the name of the CephFS
+* ``placement`` is a :ref:`orchestrator-cli-placement-spec`.
+    
+This command will 
+
+* create the required Ceph pools, 
+* create the new CephFS
+* deploy mds servers.
```

Comment:
Ditto with this one.

Reply:
`This command will create the required Ceph pools, create the new CephFS, and deploy mds servers.`

There's no reason to have it split up.